### DISCUSSION TITLE
Add the ability to name the file

### DISCUSSION THREAD

**Discussion by 3dPrintingMD** [TimPaterson/Fusion360-Batch-Post#32] [Ideas]
It would be great if there was a way to name the files. As it currently is, it will always be "1." If you put more than 1 op in a folder you have to manually rename them to avoid writing over the previous code.

**Comment by TimPaterson** [comment]
In the Introduction of the ReadMe file says "Each setup is put in a file with the name of the setup." Then there are sections labeled Subfolders and Sequence Numbers that go into much more detail on how files are named. If this isn't working as described for you, you should provide a detailed example including a screen shot of your list of setups and what post processor you're using.
